### PR TITLE
[doc] fix typo & update release-process.md for opam/docker packaging

### PR DESCRIPTION
**Pull request by erikmd**
**Kind:** documentation

Just a change in `test-suite/README.md` (that was missing in https://github.com/coq/coq/pull/8630) and a rework of one item in `dev/doc/release-process.md`, adding hyperlinks for completeness.

Depends-on: PR https://github.com/coq/coq/pull/11450

### CODE REVIEW

File: dev/doc/release-process.md [hunk 1]
```diff
@@ -126,9 +126,9 @@ in time.
 - [ ] Send an e-mail on Coqdev announcing that the tag has been put so that
   package managers can start preparing package updates (including a
   `coq-bignums` compatible version).
-- [ ] Ping **@erikmd** to update the Docker images in `coqorg/coq`
-  (requires `coq-bignums` in `extra-dev` for a beta / in `released`
-  for a final release).
+- [ ] When opening the corresponding PR in the opam repository ([`coq/opam-coq-archive`](https://github.com/coq/opam-coq-archive) or [`ocaml/opam-repository`](https://github.com/ocaml/opam-repository)),
+  the package managers can Cc **@erikmd** to anticipate the Docker release in [`coqorg/coq`](https://hub.docker.com/r/coqorg/coq)
+  (need for a `coq-bignums` opam package in [`extra-dev`](https://github.com/coq/opam-coq-archive/tree/master/extra-dev), resp. [`released`](https://github.com/coq/opam-coq-archive/tree/master/released)).
```
Comment (jfehrle) [inline dev/doc/release-process.md]: I don't understand the parenthetical comment.

Would be nice to spell out "resp.", do you mean "respectively"?

**Review by jfehrle — COMMENTED** [review]
Don't quite understand all of this, maybe you can clarify?  Not too familiar with this.


Also, the PR title seems much too generic, could you be more specific, eg something about how to build opam libraries?

File: dev/doc/release-process.md (hunk 1 above)
Comment (erikmd) [reply dev/doc/release-process.md]: @jfehrle thanks for your proofreading, indeed the grammar of the parenthetical comment wasn't very clear, I'm going to push a new version.

**Comment by erikmd** [comment]
@jfehrle 
> maybe you can clarify?

The overall idea was just to better document the steps that occur after tagging a new Coq release, so in particular the docker-coq packaging (which itself relies on the opam packaging of Coq *and* Bignums) can be prepared more smoothly (while in a couple of past releases, it took more time than expected because I was not aware of the OPAM PR for Coq when it was opened)

> Also, the PR title seems much too generic, could you be more specific, eg something about how to build opam libraries?

Done, thanks.

File: dev/doc/release-process.md [hunk 2]
```diff
@@ -126,9 +126,9 @@ in time.
 - [ ] Send an e-mail on Coqdev announcing that the tag has been put so that
   package managers can start preparing package updates (including a
   `coq-bignums` compatible version).
-- [ ] Ping **@erikmd** to update the Docker images in `coqorg/coq`
-  (requires `coq-bignums` in `extra-dev` for a beta / in `released`
-  for a final release).
+- [ ] When opening the corresponding PR for `coq` in the opam repository ([`coq/opam-coq-archive`](https://github.com/coq/opam-coq-archive) or [`ocaml/opam-repository`](https://github.com/ocaml/opam-repository)),
+  the package managers can Cc **@erikmd** to anticipate the Docker release in [`coqorg/coq`](https://hub.docker.com/r/coqorg/coq)
```
Comment (jfehrle) [inline dev/doc/release-process.md]: This makes more sense now.

"anticipate" is not quite the right word here, though.  You might just say something like "to request that he prepare the necessary Docker files"

Comment (erikmd) [reply dev/doc/release-process.md]: Thanks.
typo here? → "that he prepares"

Comment (erikmd) [reply dev/doc/release-process.md]: Nevermind, "to request that he prepare" was OK (sorry for the noise)

**Comment by erikmd** [comment]
Hi, it seems the CI build for this PR just triggered that error:

2020-01-28_22-30-55_Screenshot_GitLab_CI_error

@coq/ci-maintainers is this due to pending CI pipelines for the same `pr-11468`? (maybe someone could cancel these old pipelines at stake in https://gitlab.com/coq/coq/pipelines)

**Comment by erikmd** [comment]
@Zimmi48: to avoid the upcoming conflict with https://github.com/coq/coq/pull/11450, I've rebased #11468 upon your PR.

**Comment by Zimmi48** [comment]
Yes, we're having troubles with GitLab lately. However, for this PR it doesn't matter much since it does not change anything that gets built.

**Review by Zimmi48 — APPROVED** [review]

**Comment by Zimmi48** [comment]
Dependent PR has been merged. Merging this now.

**Pull request merged**